### ISSUE TITLE
bezierjs/bezier.js:487: ERROR - Parse error. Internet Explorer has a non-standard intepretation of trailing commas. Arrays will have the wrong length and objects will not parse at all.

### ISSUE THREAD

**Issue by yi** [Pomax/bezierjs#14]
Hi Pomax,

Great work! 

I want to use a minified version of your bezierjs and found following little issue.

It's here: https://github.com/Pomax/bezierjs/blob/gh-pages/bezier.js#L487

```
yi@localhost -> java -jar  ../../scripts/compiler.jar --js bezierjs/bezier.js --js_output_file /dev/null --compilation_level SIMPLE_OPTIMIZATIONS
bezierjs/bezier.js:487: ERROR - Parse error. Internet Explorer has a non-standard intepretation of trailing commas. Arrays will have the wrong length and objects will not parse at all.
  };
   ^

1 error(s), 0 warning(s)
```

**Comment by Pomax** [comment]
that's a flat out bug; there shouldn't be any trailing commas.

**Comment by Pomax** [comment]
fixed with v1.0.7
